phpstan/phpstan/1.0/phpstan.neon

```diff
@@ -0,0 +1,8 @@
+parameters:
+    level: 6
```

Comment:
Why 6?

Reply:
Default 0 means almost no checks. Max is 10 and it’s even too strictly. The level 6 is on their website in the getting started guide, so a new developer would probably start with that level.